docs/details/installation.md

```diff
@@ -30,7 +30,12 @@ Once you have a build, you probably want to use it in a JavaScript project. We w
 Now you will be able to use your custom-built version of bluzelle-js exactly the same as if you had downloaded it off NPM.
 
 
-## Running tests
+## Running unit tests
```

Comment:
It won't let me comment on the appropriate line, but please remove all references to BluzelleESR in this document and all others.

Reply:
resolved in new commit